MREUnityRuntime/MREUnityRuntimeLib/Patching/PatchingUtils.cs

```diff
@@ -61,6 +62,16 @@ public static string GeneratePatch(string _old, string _new)
         }
 
         public static Vector3Patch GeneratePatch(MWVector3 _old, Vector3 _new)
+        {
+            MWVector3 _mwNew = null;
+            if (_new != null)
+            {
+                _mwNew = _new.ToMWVector3();
```

Comment:
The methods ToMW*() all allocate a new MW type, and it looks like we are just doing this here to have an object to use in the comparison to generate a patch.  We should look at an alternative way to do this so that we don't have a ton of allocations just to generate patches.  See the below methods as well.

Reply:
Reverted this change. Was no longer used.

---
In reply to: [259984496](https://github.com/Microsoft/mixed-reality-extension-unity/pull/34#discussion_r259984496) [](ancestors = 259984496)